test/integration/scheduler_perf/util.go

```diff
@@ -53,27 +56,50 @@ const (
 
 var dataItemsDir = flag.String("data-items-dir", "", "destination directory for storing generated data items for perf dashboard")
 
+func newDefaultComponentConfig() (*config.KubeSchedulerConfiguration, error) {
+	gvk := v1beta1.SchemeGroupVersion.WithKind("KubeSchedulerConfiguration")
+	cfg := config.KubeSchedulerConfiguration{}
```

Comment:
I think if you pass this, you don't need to set the gvk, but you can check.

Reply:
verified, this doesn't work without gvk being set, I will dive into the `decode` method to have a double check in my next day.

Reply:
ah, makes sense, we need a version to apply the defaults